### DISCUSSION TITLE
SIte Not Building After Migrating From git submodule to go module

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [google/docsy#1880] [Q&A]
The last couple of days I've been trying to migrate my site from using Docsy via git submodule to the go module but I've been getting to following error:
```
Error: failed to load config: "/home/sam/sites/other/hockey-rules/hugo.yaml:1:1": 
failed to unmarshal YAML: yaml: unknown anchor 'desc' referenced
```

**Comment by deining** [comment]
> ```
> failed to unmarshal YAML: yaml: unknown anchor 'desc' referenced
> ```

Little context here, hard to guess what's going on here . Please be more specific here:

- Can you post your `hugo.yaml`, please, especially the section that contains the anchor `desc`.
- What is your hugo environment? Can you report back the output of `hugo env`?
- Are your site sources publicly available so that we can have a look?
